versions-maven-plugin/src/site/site.xml

```diff
@@ -26,7 +26,7 @@
       <item name="Replacing -SNAPSHOT versions with their corresponding releases" href="examples/use-releases.html"/>
       <item name="Changing the project version" href="examples/set.html"/>
       <item name="Changing aggregator project version" href="examples/setaggregator.html"/>
-      <item name="Recording Changes" href="examples/resolve-changes.html"/>
+      <item name="Recording Changes" href="/examples/recording-changes.html"/>
```

Comment:
This change is already included in my own PR #1284, but this leading `/` character is incorrect. Please remove it.